homeworks/roman.vlasov_batteryfree/homework_2/palindromicnumbers.js

```diff
@@ -0,0 +1,27 @@
+function palnum(i, j) {
+    for (let i1 = i; i1 <= j; i1++) {
+        if (i1.toString() === i1.toString().split('').reverse().join('')) {
+            console.log(i1);
+        }
+    }
+}
+const readline = require('readline');
+
+const readconsole = readline.createInterface(process.stdin, process.stdout);
+readconsole.question('Enter the number #1(>=500):', (answer) => {
+    const num1 = +answer;
+    if (typeof (num1) === 'number' && num1 >= 500) {
+        readconsole.question('Enter the number #2(<number #1):', (answer1) => {
+            const num2 = +answer1;
+            if (typeof (num2) === 'number' && num1 < num2) {
+                palnum(num1, num2);
+            } else {
+                console.log('***Invalid data entered***');
+            }
+            readconsole.close();
+        });
+    } else {
+        console.log('***Invalid data entered***');
+        readconsole.close();
```

Comment:
good work!

tip: try to follow DRY (Don't Repeat Yourself) principles as much as you can, e.g. readconsole.close() could be ran only once after you script. Pay attention to such details in your future tasks ;)